test/integration/scheduler_perf/main_test.go

```diff
@@ -56,21 +48,17 @@ func TestMain(m *testing.M) {
 	})
 	flag.CommandLine = &fs
 
-	featureGate := featuregate.NewFeatureGate()
-	runtime.Must(logsapi.AddFeatureGates(featureGate))
```

Comment:
this flow appears more in keeping with our direction than the `init` block replacement.  Why the switch here?

Reply:
`LoggingFeatureGate` and `LoggingConfig` must be exported. There are other, out-of-tree users of this package which might want to initialize logging the same way. They don't have to, but if they do, they have to use the same instances as the code in `setupTestCase`.

Reply:
While out-of-tree usage is discouraged, it's what people have to do which build custom schedulers.

Reply:
This seems likely to drift against the emulation version changes and custom schedulers with featuregates may run into problems similar to the sample-apiserver problem we hit.  Since we're willing to break golang compatibility with k/k, I suppose we can expose this and deal with a break later.